hooks/useWindowDimensions.js

```diff
@@ -1,10 +1,12 @@
 import { useState, useEffect } from 'react';
 
-export default function useWindowDimensions() {
-
-  const hasWindow = typeof window !== 'undefined';
+const initDimensions = {
+  height: 0,
+  width: 0,
+};
```

Comment:
The `initDimensions` object is a good addition. This ensures initial state is predictable.